packages/make-styles/package.json

```
@@ -17,8 +17,10 @@
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "test": "just-scripts test",
-    "test:watch": "just-scripts jest-watch"
+    "test": "jest",
+    "test:watch": "just-scripts jest-watch",
+    "docs": "api-extractor run --config=config/api-extractor.local.json --local",
+    "build:local": "tsc -p . --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output dist/make-styles/src && yarn docs"
```

to fix your issue with api-extractor:

```suggestion
    "build:local": "tsc -p . --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output dist/ && yarn docs"
```